extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -164,25 +187,24 @@ export class AmpSlideScroll extends BaseSlides {
 
   /** @override */
   onLayoutMeasure() {
-    /** @private {number} */
     this.slideWidth_ = this.getLayoutWidth();
 
-    /** @private {number} */
     this.previousScrollLeft_ = this.slidesContainer_./*OK*/scrollLeft;
   }
 
   /** @override */
   layoutCallback() {
-    if (this.slideIndex_ == null) {
+    if (this.slideIndex_ === null) {
```

Comment:
I don't understand this at all. Is there an error without it? (like unreachable code?)

That would be a compiler bug, indeed